### PR TITLE
Implemented most Android functions

### PR DESCRIPTION
**Pull request by bitsal**
encrypt and decrypt should work now.

### CODE REVIEW

**Comment by bitsal** [comment]
related to https://github.com/tectiv3/react-native-aes/issues/1

File: README.md [hunk 1]
```diff
@@ -51,18 +51,55 @@ protected List<ReactPackage> getPackages() {
...
+
+const generateKey = (password, salt) => {
+    if (Platform.OS === 'ios') {
+        return Aes.pbkdf2(password, salt);
+    }
+        
+    return new Promise((resolve, reject) => {
+        Aes.pbkdf2(password, salt, key => resolve(key), error => reject(error));
+    });
+};
```
Comment (tectiv3) [inline README.md]: Are you sure you need this promise wrapper for android platform?
It should work equally on both platforms.

File: README.md [hunk 2]
```diff
@@ -51,18 +51,55 @@ protected List<ReactPackage> getPackages() {
...
+
+const encrypt = (text, keyBase64) => {
```
Comment (tectiv3) [inline README.md]: Same here, I think it is a needless wrapper.

File: README.md (hunk 1 above)
Comment (bitsal) [inline README.md]: Will verify and fix it next week.

**Comment by tectiv3** [comment]
Thank you!

**Pull request merged**